ostelco-core/Models/StageDecider.swift

```diff
@@ -19,6 +19,7 @@ public class LocalContext {
     public var hasSeenVerifyIdentifyOnboarding: Bool
     public var selectedVerificationOption: IdentityVerificationOption?
     public var hasSeenLocationPermissions: Bool
+    public var simProfile: SimProfile?
```

Comment:
Why are we adding the sim profile to the local context when the Context already has it?

Reply:
In a multi device world, we don't really know which sim profile from the server context we should use. There could be a profile ready for download that was initiated from a different device etc etc.
To be on the safe side, we always provision a new sim profile if you enter the esim instructions flow, at least until we figure out which cases it's safe to use an existing sim profile from server context.